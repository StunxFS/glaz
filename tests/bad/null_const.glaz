const CONST_X: *i32 = null;

fn main() { }
