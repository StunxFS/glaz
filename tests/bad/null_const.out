tests/bad/null_const.glaz:1:8: error: cannot assign null value to constant `CONST_X`
     | 
   1 | const CONST_X: *i32 = null;
     |       ^^^^^^^
glaz error: could not compile `null_const`, aborting due to previous error.
